scripts/pylintrc_default

```diff
@@ -322,7 +322,7 @@ no-docstring-rgx=__.*__
 # DEFAULT:  ignored-modules=
 # RATIONALE:  six aliases stuff for compatibility.
 #             google.protobuf fixes up namespace package "late".
-ignored-modules = six, google.protobuf
+ignored-modules = six, google
```

Comment:
I'd just as soon disable their import checks altogether:  we *know* that the imports are good, because coverage.